app/styles/_variables.scss

```diff
@@ -309,6 +309,9 @@ $overlay-background-color: rgba(0, 0, 0, 0.4);
   --nudge-arrow-z-index: 16;
   --drag-overlay-z-index: 15;
   --side-panel-z-index: 14;
+  // We want second level to fall under top level when scrolling
+  --list-sticky-header-top-level: 11;
+  --list-sticky-header-second-level: 10;
```

Comment:
Nit: should we suffix these with `-z-index` for more clarity? Or mention z-index in the comment above?